### PR TITLE
{tools}[foss/2018a] mordecai v2.0.1

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
A code for full text geoparsing as a Python library.

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 5 out of 8 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/d3d355948846edba1bd0b8307f1f4167 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/3401099dfd8d34bcf5081a020643d688 for a full test report.

**Comment by verdurin** [comment]
Hmm. This happens:
```
== 2018-08-15 12:23:50,646 easyconfig.py:1215 WARNING Unable to resolve template value https://pypi.python.org/packages/source/%(nameletter)s/%(name)s with dict {'version': '2.0.1', 'name': 'mordecai', 'options': {'checksums': ['c417f343eac29eccc8af0b5cacb47da8d17bfc4dd4187d3fa193828622b5372c', 'a52cf0b3ea4ac61c250a011c61a186644b2ade8f2adefc1560654d03c574dccf'], 'patches': ['mordecai-2.0.1_fix_bad_path_handling.patch'], 'source_urls': ['https://pypi.python.org/packages/source/%(nameletter)s/%(name)s']}}
```
which then means the downloads fail.

**Comment by akesandgren** [comment]
Strange, I don't remember having any problems with that

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.5.1804, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/29e95fc9031126cb994c35ffdf9a1634 for a full test report.

File: easybuild/easyconfigs/m/mordecai/mordecai-2.0.1-foss-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+information."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_download_dep_fail = True
+
+pyver = '3.6.4'
+pymajmin = ''.join(pyver.split('.')[:2])
```
Comment (boegel) [inline easybuild/easyconfigs/m/mordecai/mordecai-2.0.1-foss-2018a-Python-3.6.4.eb]: @akesandgren Minor cleanup: `pymajmin` isn't used anywhere, so just remove?

And then there's no need for `pyver` anymore either, just specify `'3.6.4'` directly in `dependencies` for `'Python'`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (1 easyconfigs in this PR)
node2459.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/75fcc0fd2b6156e29dc8eb1f326ec5d1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 12 out of 12 (1 easyconfigs in this PR)
node3123.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/859407811e722d2f89db1996d6dd4852 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**